lms/envs/common.py

```diff
@@ -2118,14 +2118,16 @@
 
 # JWT Settings
 JWT_AUTH = {
-    'JWT_SECRET_KEY': None,
+    # TODO Set JWT_SECRET_KEY to a secure value. By default, SECRET_KEY will be used.
+    # 'JWT_SECRET_KEY': '',
```

Comment:
In configuration, yes: https://github.com/edx/configuration/blob/8710d111c2db14e9aeccc3f5cfaf989905cfe541/playbooks/roles/edxapp/defaults/main.yml#L779-L782.